packages/preact-iso/router.js

```diff
@@ -1,16 +1,17 @@
 import { h, createContext, cloneElement } from 'preact';
 import { useContext, useMemo, useReducer, useEffect, useLayoutEffect, useRef } from 'preact/hooks';
 
-const UPDATE = (state, url, push) => {
+const UPDATE = (state, url) => {
+	let push = true;
```

Comment:
How can we get to the `else if (push === false) history.replaceState(null, '', url);` now?

Reply:
We cannot. however we never could (the `UPDATE` method is not exported, just the `route(url)` is). Now it's just obvious to notice.
Please read this PR description for explanation and try to get to evaluate `history.replaceState` on master branch if you can.

Reply:
Right, #413 will provide a way to use `push`.